website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1121 @@ update your API calls accordingly.
+the newer `/pki/issuer/:issuer_ref/crl` path; the latter correctly places the
+PEM-encoded CRL in the `crl` field whereas the former incorrectly places it
+in the `certificate` field.
 
-May optionally append additional CA certificates. Useful when creating an
-intermediate CA to ensure a full chain is returned when signing or generating
-certificates.
+These are unauthenticated endpoints.
 
-Not needed if you are generating a self-signed root certificate, and not used
-if you have a signed intermediate CA certificate with a generated key (use the
-`/pki/intermediate/set-signed` endpoint for that). _If you have already set a
-certificate and key, they will be overridden._
+| Method | Path                              | Issuer    | Format |
+| :----- | :-------------------------------- | :-------- | :----- |
+| `GET`  | `/pki/cert/crl`                   | `default` | JSON   |
+| `GET`  | `/pki/crl`                        | `default` | DER    |
+| `GET`  | `/pki/crl/pem`                    | `default` | PEM    |
+| `GET`  | `/pki/issuer/:issuer_ref/crl`     | Selected  | JSON   |
+| `GET`  | `/pki/issuer/:issuer_ref/crl/der` | Selected  | DER    |
+| `GET`  | `/pki/issuer/:issuer_ref/crl/pem` | Selected  | PEM    |
 
-| Method | Path             |
-| :----- | :--------------- |
-| `POST` | `/pki/config/ca` |
+#### Parameters
 
-### Parameters
+- `issuer_ref` `(string: <required>)` - Reference to an existing issuer,
+  either by Vault-generated identifier, the literal string `default` to
+  refer to the currently configured default issuer, or the name assigned
+  to an issuer. This parameter is part of the request URL.
 
-- `pem_bundle` `(string: <required>)` – Specifies the unencrypted private key
-  and certificate, concatenated in PEM format.
+~> Note: This parameter is not present on the `/pki/cert/crl` and
+   `/pki/crl(/pem)?` paths and takes the implicit value `default`.
 
-### Sample Request
+#### Sample Request
 
 ```shell-session
 $ curl \
-    --header "X-Vault-Token: ..." \
-    --request POST \
-    --data "@payload.json" \
-    http://127.0.0.1:8200/v1/pki/config/ca
+    http://127.0.0.1:8200/v1/pki/issuer/root-x1/crl
 ```
 
-Note that if you provide the data through the HTTP API it must be
-JSON-formatted, with newlines replaced with `\n`, like so:
+#### Sample Response
 
 ```json
 {
-  "pem_bundle": "-----BEGIN RSA PRIVATE KEY-----\n...\n-----END CERTIFICATE-----"
+  "data": {
+    "crl": "-----BEGIN X509 CRL-----\nMIIBizB1AgEBMA0GCSqGSIb3DQEBCwUAMBIxEDAOBgNVBAMTB3Jvb3QgeDEXDTIy\n..."
+  }
 }
 ```
 
-## Read CRL Configuration
+### List Certificates
 
-This endpoint allows getting the duration for which the generated CRL should be
-marked valid. No CRL configuration will be returned until the configuration is
-set, but the CRL will still default to enabled with 72h expiration.
+This endpoint returns a list of the current certificates by serial number only.
+The response does not include the special serial numbers (`ca`, `ca_chain`,
```

Comment:
Is this true, I thought we wrote out the serial number of root generated ca certificates to /certs and returned it with this API? Or am I misunderstanding what you mean by "special serial numbers"?

Reply:
Also we might want to mention this is cluster specific?

Reply:
The special serial numbers are the ones listed in parenthesis afterwards :-) So `ca`'s serial is a "regular" serial number, but `ca` or `crl` or `ca_chain` are special values (e.g., `/pki/cert/ca` or `/pki/cert/crl` or `/pki/cert/ca_chain`) that behaves differently than the other serial numbers.